kernel-impl/src/main/java/org/sakaiproject/user/impl/BaseUserDirectoryService.java

```diff
@@ -1973,6 +1971,39 @@ public UserEdit newUser(String eid)
 		return u;
 	}
 
+	public void updateUserId(String id,String newEmail)
+	{
+		try {
+			List<String> locksSucceeded = new ArrayList<String>();
+			// own or any
+			List<String> locks = new ArrayList<String>();
+			locks.add(SECURE_UPDATE_USER_ANY);
+			locks.add(SECURE_UPDATE_USER_OWN);
+			locks.add(SECURE_UPDATE_USER_OWN_EMAIL);
+			locksSucceeded = unlock(locks, userReference(id));
+
+			if(locksSucceeded.contains(SECURE_UPDATE_USER_ANY) || locksSucceeded.contains(SECURE_UPDATE_USER_OWN_EMAIL )|| locksSucceeded.contains(SECURE_UPDATE_USER_OWN)) {
```

Comment:
Why would this contain anything else? Can't you just check that `locksSucceeded` is not empty?